src/benchmark.jl

```diff
@@ -272,36 +268,26 @@ end
   end
 
   # Get a cache
-  if !haskey(setups[1],:dts)
-    sol = solve(_prob,setups[1][:alg];
-          kwargs...,setups[1]...,
-          abstol=abstols[1],
-          reltol=reltols[1],
-          timeseries_errors=false,
-          dense_errors = false)
-  else
-    sol = solve(_prob,setups[1][:alg];
-          kwargs...,setups[1]...,abstol=abstols[1],
-          reltol=reltols[1],dt=setups[1][:dts][1],
-          timeseries_errors=false,
-          dense_errors = false)
-  end
+  _abstols = get(setups[1],:abstols,abstols)
+  _reltols = get(setups[1],:reltols,reltols)
+  _dts = get(setups[1],:dts,zeros(length(_abstols)))
```

Comment:
Is zero `dt` the sentinel value for nonspecified `dt`?

Reply:
yes